include/seastar/http/routes.hh

```diff
@@ -90,6 +90,9 @@ public:
      * @param url the url to match (note that url should start with /)
      * @param handler the desire handler
      * @return it self
+     * @attention If successful, this method takes ownership of the handler's
+     * pointer. It will be deleted upon `routes` instance destruction. If the
+     * method throws, it is up to caller to take care of the handler pointer.
```

Comment:
```suggestion
     * @attention If successful, this method takes ownership of the handler
        pointer. It will be automatically deleted when the routes instance is
        destroyed. However, if the method throws, the caller is responsible
        for deleting the handler pointer.
```

more specific this way, and easier to digest.